packages/rollup/index.js

```diff
@@ -239,6 +239,6 @@ async function main(args) {
   }
 }
 
-if (require.main === module) {
-  main(process.argv.slice(2)).then(r => (process.exitCode = r));
-}
+if (require.main == module) {
+  (async () => process.exitCode = await main(process.argv.slice(2)))();
```

Comment:
Does this cause any actual change? How about the `==` change above?

Reply:
no, just style to match other files while I was here